2227/2/lib/MostObsceneRappersFinder.rb

```diff
@@ -0,0 +1,10 @@
+# Data Analysis
+class MostObsceneRappersFinder
+  def self.stats_of_rappers(number)
+    data = DataStorage.new
```

Comment:
do u really need this here? 
Pass rapers array instead and work with it. Avoid extra dependencies between classes